modules/learn/pages/security/authentication-overview.adoc

```diff
@@ -77,6 +77,34 @@ For more information on the using the REST API to configure security settings, s
 
 For cases that do not use SCRAM-SHA, such as those that implement TLS network encryption, it is recommended to _disable_ all variations of SCRAM-SHA; as this will prevent the SCRAM-SHA PBKDF2 hashes from being generated, when Argon2ID provides stronger security for those credentials.
 
+
+[#password-hash-migration]
+=== Automatic Password Hash Migration
+
+You can select which algorithm Couchbase Server uses to hash locally stored passwords by setting `passwordHashAlg` using the `/settings/security` REST API. 
+When you change the hash algorithm Couchbase Server uses, it does not update all of the stored passwords using the new algorithm.
+It cannot rehash the existing passwords as it does not have access to the original, unhashed password.
+Instead, existing passwords remain hashed using the previous hashing algorithm.
+Couchbase Server uses the new hashing algorithm for newly created passwords for either new accounts or when an existing account changes its password. 
+Users can authenticate using passwords hashed with any hashing algorithm Couchbase Server supports.
```

Comment:
To future proof this, Maybe remove the word "new" here: Couchbase SErver uses the new hashing algorithm..."